pyproject.toml

```diff
@@ -8,12 +8,16 @@ documentation = "https://github.com/bee-san/pyWhat/wiki"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2"
```

Comment:
What happens between .6 and .6.2 that we need this? :)

Reply:
black requires python 3.6.2 or higher @bee-san

Reply:
Then add it under the dev dependencies so that people can still use Python `3.6.0` if they want.

Reply:
Yeah I agree with Skeletal here, REMnux requires 3.6 :shrug: :(